content-api/content-service/conf/application.conf

```diff
@@ -743,4 +743,7 @@ indexHtmlValidation.env=false
 hierarchyUpdate.allow.resource.at.root.level=false
 
 #timeout
-actor.timeoutMillisec = 120000
\ No newline at end of file
+actor.timeoutMillisec = 120000
+
+# Artifact Url allowed for copy resource
+content.copy.is_copy_artifacturl= true
```

Comment:
why this value is set to "true" -- in that case, it is just going to execute the previous code only?
when this would be "false" and what will be result of modified code?

Reply:
This will be true for diksha, but in our case we made it false. And when it is false then the artifacturl will not be copied during copying the course content both for the complete course level and resource level.